app/lib/features/news/widgets/news_item_slide/video_slide.dart

```diff
@@ -40,33 +34,47 @@ class VideoSlide extends StatelessWidget {
   Widget build(BuildContext context) {
     return Container(
       key: NewsUpdateKeys.videoNewsContent,
-      color: bgColor,
-      alignment: Alignment.center,
-      child: FutureBuilder<File>(
-        future: getNewsVideo(),
-        builder: (BuildContext context, AsyncSnapshot<File> snapshot) {
-          if (snapshot.hasError) {
-            _log.severe(
-              'Failed to load video of slide',
-              snapshot.error,
-              snapshot.stackTrace,
-            );
-            return Center(
-              child: Text(L10n.of(context).loadingFailed(snapshot.error!)),
-            );
-          }
+      child: renderVideoContent(),
+    );
+  }
+
+  Widget renderVideoContent() {
```

Comment:
this could go up being reusable also for video content rendering

Reply:
Same as above.